src/main/java/de/tum/in/www1/artemis/service/LectureUnitProcessingService.java

```diff
@@ -154,7 +156,7 @@ public LectureUnitInformationDTO getSplitUnitData(MultipartFile file) {
                     .map(lectureUnitSplit -> new LectureUnitSplitDTO(lectureUnitSplit.unitName, ZonedDateTime.now(), lectureUnitSplit.startPage, lectureUnitSplit.endPage))
                     .toList();
             // return units information, maximum number of pages and by default remove break slides and remove solution slides are false
-            return new LectureUnitInformationDTO(units, numberOfPages, false, false);
+            return new LectureUnitInformationDTO(units, numberOfPages, false, "", false, "");
```

Comment:
To reduce data you might want to set the empty Strings to null and add `@JsonInclude(Include. NON_NULL)` annotation to the DTO.

Reply:
The DTO is returned to the client. See: `getAttachmentUnitsData` endpoint.

Reply:
You are right and the client doesn't try to resolve the value, so I won't need to check for `null` there. Usage: https://github.com/ls1intum/Artemis/blob/ab64bc90c899224e528419c9315ea3197c83c653/src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.ts#L85

6670f789049f93a0c33f7f05af92ed8cb0f0a4fb